utilities/helpers.scss

```diff
@@ -91,4 +91,22 @@
   color: $color;
 }
 
+@mixin show-custom-scroll() {
+  ::-webkit-scrollbar {
+    width: 11px;
+  }
+  
+  ::-webkit-scrollbar-track {
+    background: var(--hdd-color-grey-light);
+  }
+  
+  ::-webkit-scrollbar-thumb {
+    width: 11px;
+    border-radius: 5px;
+    background: var(--hdd-color-grey);
+    border-left: 2px solid var(--hdd-color-grey-light);
+    border-right: 2px solid var(--hdd-color-grey-light);
+  }
+}
+
 
```

Comment:
Bør vi ha løsning for ::-webkit-scrollbar-corner?

Reply:
Selve tracken er tykkere en thumben så tror det blir rart å endre farge der, eller hva tenker du på?